Command/KlarnaShippingCommand.php

```diff
@@ -163,18 +162,15 @@ public function execute(InputInterface $input, OutputInterface $output)
                     $mollieOrder = $this->apiClient->orders->get($transaction->getMollieId());
 
                     if ($mollieOrder === null) {
-                        $io->note('No Order found in Mollie for this transaction!');
-
-                        $this->logger->error('Klarna Ship Command: No order found in Mollie for transaction: ' . $transaction->getId() . '!');
+                        $tableView->addRow([$transaction->getOrderNumber(), '', 'no order found in Mollie']);
 
                         $countFailed++;
                         continue;
                     }
 
                     # REPAIR ORDERS ALREADY SHIPPED ----------------------------------------------------------------
-                    if (count($mollieOrder->shipments()) > 0) {
-                        $io->note('Order already shipped!');
-                        $io->text('The order is already shipped in Mollie and must not be shipped again! Repairing this order!');
+                    if ($mollieOrder->shipments()->count() > 0) {
+                        $tableView->addRow([$transaction->getOrderNumber(), '', 'order is already shipped in Mollie', $order->getShop()->getName()]);
```

Comment:
bitte repairing mitaufnehmen im neuen text, sowie das "must not be shipped again"